docs/guides/cd.md

```diff
@@ -1,64 +1,64 @@
 # Continuous Deployment of your Functions
 
-To continously deploy your functions on TriggerMesh you can register a source code repository which contains your function code and a manifest (`serverless.yaml`).
 
-The **_Repositories_** section in the console contains a view to register any repository, from previously linked accounts, for continuous deployment.
 
+The Triggermesh console provides an interface called [Repositories.](https://cloud.triggermesh.io/function-triggers) Here the user is able to select a codebase, from a linked repository provider, to be continuously deployed. This codebase **must** contain a deployment manifest[^1] . 
 
-## Register the repository
 
+## Register the repository
 
-**Begin by forking the [sample repository](https://github.com/sebgoa/tmserverless) which contains a function and an example `serverless.yaml` manifest.**
+###### The following example uses this [sample repository.](https://github.com/sebgoa/tmserverless) To follow along begin by forking the example to your repository provider. 
 
-Select the _Repositories_ section from the Navigation menu and then _CREATE NEW_ 
+1. Select the _Repositories_ section from the Navigation menu and then _CREATE NEW_ 
 
 ![](../images/tmRepoView.png)
 
-Select a source control provider. Upon submission of this form you will be asked to authenticate with the providor chosen in the previous step.
+2. Select a source control provider. Upon submission of this form you will be asked to authenticate with the chosen provider.
```

Comment:
A neat trick with markdown is that you can just restate `1.` and it should automagically do the right thing with ordering.

Reply:
oh cool! thank you!